core/src/main/scala/org/apache/spark/scheduler/HealthTracker.scala

```diff
@@ -64,6 +65,7 @@ private[scheduler] class HealthTracker (
   val EXCLUDE_ON_FAILURE_TIMEOUT_MILLIS = HealthTracker.getExludeOnFailureTimeout(conf)
   private val EXCLUDE_FETCH_FAILURE_ENABLED =
     conf.get(config.EXCLUDE_ON_FAILURE_FETCH_FAILURE_ENABLED)
+  private val decommission = conf.get(config.EXCLUDE_ON_FAILURE_DECOMMISSION_ENABLED)
```

Comment:
Shall we keep the same pattern of naming here? e.g., `EXCLUDE_ON_FAILURE_DECOMMISSION_ENABLED`. Usually, it's good for developers to understand the code here.